build/PlatformInstaller.targets

```diff
@@ -0,0 +1,300 @@
+    <!-- RH hosting bundle -->
+    <MSBuild Projects="$(MSBuildProjectFullPath)" Targets="_GenerateRpm" Properties="$(Hosting_RPM_Arguments)" />
+
+    <!-- Remove Docker Image to save disk space -->
+    <MSBuild Projects="$(MSBuildProjectFullPath)" Targets="_RemoveDockerImage" Properties="Image=$(Image)" />
+  </Target>
+
+  <Target Name="RunDebTool">
+    <!-- Install dotnet-deb tool -->
+    <MSBuild Projects="$(_DebToolDir)dotnet-deb-tool-consumer.csproj" Targets="Restore" />
+
+    <!-- Build deb package -->
+    <Exec
+      Command="dotnet deb-tool -i $(_WorkLayoutDir) -o $(_WorkOutputDir) -n $(INSTALLER_NAME) -v $(INSTALLER_VERSION)"
+      WorkingDirectory="$(_DebToolDir)" />
+  </Target>
+
+  <Target Name="_GenerateDeb">
+    <!-- Create layout: Clear work directory -->
+    <RemoveDir Directories="$(_WorkRoot)" />
+    <MakeDir Directories="$(_WorkRoot)" />
+
+    <!-- Create layout: Extract archive if given -->
+    <MakeDir Directories="$(_WorkLayoutDir)package_root\" />
+    <Exec Command="tar -xzf $(SFArchive) -C $(_WorkLayoutDir)package_root/" Condition="'$(SFArchive)'!=''" />
+
+    <!-- Create layout: Generate and Place debian_config.json -->
+    <Copy
+      SourceFiles="$(DebConfigFile)"
+      DestinationFiles="$(_WorkLayoutDir)debian_config.json"
+      OverwriteReadOnlyFiles="True"
+      SkipUnchangedFiles="False"
+      UseHardlinksIfPossible="False" />
+
+    <ItemGroup>
+      <DebConfigItems Include="MAINTAINER_NAME" Replacement="$(MaintainerName)" />
+      <DebConfigItems Include="MAINTAINER_EMAIL" Replacement="$(MaintainerEmail)" />
+      <DebConfigItems Include="HOMEPAGE" Replacement="$(Homepage)" />
+      <DebConfigItems Include="INSTALL_ROOT" Replacement="$(InstallRoot)" />
+      <DebConfigItems Include="PACKAGE_NAME" Replacement="$(DebPrefix)" />
+      <DebConfigItems Include="PACKAGE_REVISION" Replacement="$(PackageRevision)" />
+      <DebConfigItems Include="LICENSE_TYPE" Replacement="$(LicenseType)" />
+      <DebConfigItems Include="SHORT_DESCRIPTION" Replacement="$(DebSummary)" />
+      <DebConfigItems Include="LONG_DESCRIPTION" Replacement="$(DebDescription)" />
+      <DebConfigItems Include="DEBIAN_DEPENDENCIES" Replacement="$(DebDependencies)" />
+    </ItemGroup>
+
+    <!-- Update versions -->
+    <RepoTasks.ReplaceInFile Filename="$(_WorkLayoutDir)debian_config.json" Items="@(DebConfigItems)" />
+
+    <!-- Build SF and Hosting Bundle Deb package -->
+
+    <!--
+      Note: KOREBUILD_DOTNET_FEED_CREDENTIAL is intentionally NOT an MSBuild variable.
+      MSBuild doesn't to the substitution correctly because the string contains %,
+      so we'll let bash do it instead.
+    -->
+
+    <Exec Command="docker run
+      --rm
+      -v $(RepositoryRoot):$(_DockerRootDir)
+      -e DOTNET_SKIP_FIRST_TIME_EXPERIENCE=true
+      -e INSTALLER_NAME=$(DebPrefix)-$(DebVersion)
+      -e INSTALLER_VERSION=$(DebVersion)
+      -e 'KOREBUILD_DOTNET_VERSION=$(KOREBUILD_DOTNET_VERSION)'
```

Comment:
We plan to deprecate these variables completely. It would be better to switch to variables as defined in `dependencies.props`.